configs/defs/rocky.textproto

```diff
@@ -0,0 +1,1179 @@
+      "critical the user re-authenticate."
+    remediation:
+      "Configure the operating system to require users to reauthenticate for privilege escalation.\n"
+      "Based on the outcome of the audit procedure, use visudo -f <PATH TO FILE> to edit the "
+      "relevant sudoers file.\n"
+      "Remove any occurrences of !authenticate tags in the file(s)."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "         path:\"/etc/sudoers\""
+      "       }"
+      "      }"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/sudoers.d/\""
+      "       }"
+      "      }"
+      "    content_entry:{"
+      "        match_type: NONE_MATCH"
+      "        match_criteria: {"
+      "          filter_regex: \"^[^#].*\\\\!authenticate\""
+      "        }"
+      "      }"
+      "    }"
+      "  }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "sudo-authentication-timeout-is-configured-correctly"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure sudo authentication timeout is configured correctly"
+    description:
+      "sudo caches used credentials for a default of 5 minutes.\nThis is for ease of use when there " 
+      "are multiple administrative tasks to perform. The timeout canbe modified to suitlocal "
+      "security policies."
+    rationale:
+      "Setting a timeout value reduces the window of opportunity for unauthorized privileged "
+      "access to another user."
+    remediation:
+      "If the currently configured timeoutis larger than 15 minutes, edit the file listed in the audit "
+      "sectionwith visudo -f <pATH TO FILE>and modifythe entry timestamp timeout=to 15 "
+      "minutes or less as per your site policy. The value is in minutes.\nThis particular entry may "
+      "appear on its own, or on the sameline as env_reset. See the following two examples:\n"
+      "Defaults env_reset, timestamp timeout=15\n"
+      "Defaults timestamp_ timeout=15\n"
+      "Defaults env_reset\n"
+      "References:\n"
+      "1. https://www.sudo.ws/man/1.9.0/sudoers.man.html"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "         path:\"/etc/sudoers\""
+      "       }"
+      "      }"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/sudoers.d/\""
+      "       }"
+      "      }"
+      "    content_entry:{"
+      "        match_type: NONE_MATCH"
+      "        match_criteria: {"
+      "          filter_regex: \"Authentication timestamp timeout:.*\""
```

Comment:
The audit doc also says to check for "timestamp_timeout=" first, and only check the authentication timestamp if the other one is not set.